test/src/hooks/use-form.test.js

```diff
@@ -43,7 +43,8 @@ describe('useForm hook', () => {
     const { result } = renderHook(() => useForm({
       initialValues: {},
       jsonSchema: { required: ['foo'] },
-      onSubmit: () => {}
+      onSubmit: () => {},
+      validationOptions: { strict: false }
```

Comment:
Please add `type: 'object'` to the JSON schema instead.

Same thing to the others in `test/src/utils/validate.test.js`.